vendor/index.html

```diff
@@ -1,9 +1,34 @@
 <!DOCTYPE html>
```

Comment:
Why is this in the /vendor folder? Are you getting your index from somewhere else?